routing/base/routing_result.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "routing/base/astar_weight.hpp"
+
+#include <vector>
+
+namespace routing
+{
+template <typename VertexType, typename WeightType>
+struct RoutingResult final
+{
+  std::vector<VertexType> m_path;
```

Comment:
By Google C++ style guide, please, keep all fields in the end of the struct.

Reply:
done